translations/messages.pt_BR.xlf

```diff
@@ -97,7 +97,7 @@
       </trans-unit>
       <trans-unit id="ebTjBb9" resname="my_times">
         <source>my_times</source>
-        <target>Meus tempos</target>
+        <target state="translated">Meus horários</target>
```

Comment:
@wtuemura are you a Kimai user? I saw these changes (thank you for helping out with the translations 👍), as you replaced "de trabalho" with "trabalhadas" below in keys and and I became curious. 

I don't speak Portuguese, so I tried G-Translate and "Meus tempos" does match the expected translation, while "Meus horários" translates to "my schedules", which would be wrong. 

That's why I am asking if you are a Kimai user => some translations are obviously much harder to understand without context. The screen "My times" opens a list of all my logged working times and not something like scheduled calendar entries.

These changes are currently blocking a merge and I would like to resolve that issue. Would be great if you could give me your opinion on that.

Reply:
Clarified at hosted.weblate.org